WMFComponents/Sources/WMFComponents/Components/Year in Review/WMFYearInReviewScrollView.swift

```diff
@@ -51,30 +45,16 @@ public struct WMFYearInReviewScrollView: View {
         ScrollView(showsIndicators: true) {
             VStack(spacing: 16) {
                 ZStack {
-                    Image(imageName, bundle: .module)
+                    Image(gifName, bundle: .module)
                         .resizable()
                         .scaledToFit()
                         .frame(maxWidth: .infinity)
-                        .ignoresSafeArea()
-                        .padding(.horizontal, 0)
                         .accessibilityHidden(true)
-
-                    if let imageOverlay {
-                        if let imageOverlayAccessibilityLabel {
-                            Image(imageOverlay, bundle: .module)
-                                .accessibilityLabel(imageOverlayAccessibilityLabel)
-                        } else {
-                            Image(imageOverlay, bundle: .module)
-                                .accessibilityHidden(true)
-                        }
-                    }
-
-                    if let overlayText = textOverlay {
-                        Text(overlayText)
-                            .font(Font(WMFFont.for(.xxlTitleBold)))
-                            .foregroundColor(.white)
-                    }
+                    WMFGIFfImageView(gifName)
+                        .aspectRatio(1.5, contentMode: .fit)
+                        .frame(maxWidth: .infinity)
                 }
+                .accessibilityLabel(altText)
```

Comment:
It's reading "WMFComponents WMFComponents bundle, image" to me for all the gifs. Let me know if you can repro. So far with my tiny tweaks I haven't been able to fix it, you may have better luck.